docs/source/rdump.rst

```diff
@@ -0,0 +1,115 @@
+rdump
+-----
+
+By default target-query generates records. Records are binary representations of artefacts that
+appear like this if they are transformed to text by the default mechanism:
+
+.. code-block:: console
+
+    <record key1="value1" key2="value2" >
+
+With rdump you can transform the stream of records to your liking. The rdump utility allows you to:
+
+* select certain fields from the records
+* filter certain records
+* create additional derived fields
+* limit the output
+* format the results
+* write the results through an adapter
+
+.. hint::
+
+    Don't know yet what a record is? Read more :ref:`here <overview/index:records>` for a detailed explanation.
+
+Field Selection
+~~~~~~~~~~~~~~~
+
+Imagine we're going to extract a list of users from a forensic image using:
+
+.. code-block:: console
+
+    $ target-query host.img -f users 
+
+To select only the name and home fields we use the ``-F`` option:
+
+.. code-block:: console
+
+    $ target-query host.img -f users | rdump -F name,home
+
+Using the ``-X`` option we can exclude fields:
+
+    $ target-query host.img -f users | rdump -X hostname,domain,sid
```

Comment:
I would love to see the output of all three commands. This will make it for the reader way easier to understand the implications of the Field selections.

Reply:
Whilst reading further I would say this comment applies to this whole file

Reply:
No, I don't think that is necessary. It also creates too much visual clutter.